db/app.js

```diff
@@ -1,9 +1,23 @@
 const express = require("express")
 const app = express()
-const { apiHealthCheck , getTopics } = require("./controller")
-
+app.use(express.json());
+const {
+    apiHealthCheck,
+    getTopics,
+    getArticleById } = require("./controller")
 app.get('/api', apiHealthCheck)
 
 app.get('/api/topics', getTopics)
 
+app.get('/api/articles/:article_id', getArticleById)
+
+app.use((err, req, res, next) => {
+    console.log(err.msg)
+    if (err.status) {
```

Comment:
Have a look at the error handling notes to set this up in a more 'separation of concerns' way!